### PR TITLE
Cleanup strings in samples

### PR DESCRIPTION
**Pull request by kimsey0**
- Normalize quote types to use double quotes everywhere. (The existing code was split almost 50/50 between single and double quotes and I couldn't find a style guide, so I choose the standard the TypeScript team uses. If you prefer single quotes, I'll just change it over to that.)
- Use template strings where possible instead of concatenating strings.

@damccorm: I finally got around to this. Could you have a look?

### CODE REVIEW

File: samples/git.ts [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-import * as GitInterfaces from 'azure-devops-node-api/interfaces/GitInterfaces';
+import * as GitApi from "azure-devops-node-api/GitApi";
+import * as GitInterfaces from "azure-devops-node-api/interfaces/GitInterfaces";
 
 export async function run() {
     let webApi: nodeApi.WebApi = await common.getWebApi();
     let gitApiObject: GitApi.IGitApi = await webApi.getGitApi();
 
-    common.banner('Git Samples');
-    let project: string = common.getProject();
```
Comment (kimsey0) [inline samples/git.ts]: And this one.

File: samples/wiki.ts [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+import * as WikiApi from "azure-devops-node-api/WikiApi";
+import * as WikiInterfaces from "azure-devops-node-api/interfaces/WikiInterfaces";
 
 export async function run() { 
     const webApi: nodeApi.WebApi = await common.getWebApi();
     const wikiApiObject: WikiApi.IWikiApi = await webApi.getWikiApi();
     const coreApiObject: CoreApi.ICoreApi = await webApi.getCoreApi();
 
-    common.banner('Wiki Samples');
-    const project: string = common.getProject();
```
Comment (kimsey0) [inline samples/wiki.ts]: And this.

File: samples/wiki.ts [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-    console.log('Project:', project);
+    console.log("Project:", project);
 
-    common.heading('Get all wikis');
+    common.heading("Get all wikis");
     const wikis: WikiInterfaces.WikiV2[] = await wikiApiObject.getAllWikis(project);
     console.log("Wikis", wikis.map((wiki) => wiki.name));
 
     let wikiId: string;
-    const createNewWiki: boolean = (wikis.length === 0);
```
Comment (kimsey0) [inline samples/wiki.ts]: And this.

**Comment by damccorm** [comment]
Looks great, thanks for cleaning this up!

**Pull request merged**